src/adapters/refract/transformTransactions.coffee

```diff
@@ -20,93 +16,80 @@ module.exports = (transactions, options) ->
+    httpRequestElement = minim.serialiser06.serialise(httpRequest)
+    httpResponseElement = minim.serialiser06.serialise(httpResponse)
 
     # In refract, we have method only in this place
-    method = _.chain(httpRequest).get('attributes.method', '').contentOrValue().value()
+    method = httpRequest.method?.toValue() or ''
 
     # Request retrieving
-    httpRequestBody = _(httpRequest).messageBodies().first()
-    httpRequestBodySchemas = _(httpRequest).messageBodySchemas().first()
+    httpRequestBody = httpRequest.messageBody
+    httpRequestBodySchema = httpRequest.messageBodySchema
     httpRequestDescription = getDescription(httpRequest, options)
-    httpRequestBodyDataStructures = _.dataStructures(httpRequest)
+    requestAttributes = httpRequest.dataStructure
 
-    if _.isEmpty(httpRequestBodyDataStructures)
-      requestAttributes = undefined
-    else
-      requestAttributes = httpRequestBodyDataStructures[0]
+    if requestAttributes
+      requestAttributes = minim.serialiser06.serialise(requestAttributes)
 
     # Response retrieving
-    httpResponseBody = _(httpResponse).messageBodies().first()
-    httpResponseBodySchemas = _(httpResponse).messageBodySchemas().first()
+    httpResponseBody = httpResponse.messageBody
+    httpResponseBodySchema = httpResponse.messageBodySchema
     httpResponseDescription = getDescription(httpResponse, options)
-    httpResponseBodyDataStructures = _.dataStructures(httpResponse)
+    responseAttributes = httpResponse.dataStructure
 
-    if _.isEmpty(httpResponseBodyDataStructures)
-      responseAttributes = undefined
-    else
-      responseAttributes = httpResponseBodyDataStructures[0]
+    if responseAttributes
+      responseAttributes = minim.serialiser06.serialise(responseAttributes)
 
     # Example Id handling
     alreadyUsedRequest = _.some(prevRequests, (prevRequest) ->
-      _.isEqual(prevRequest, httpRequest)
+      _.isEqual(prevRequest, httpRequestElement)
     )
 
     alreadyUsedResponse = _.some(prevResponses, (prevResponse) ->
-      _.isEqual(prevResponse, httpResponse)
+      _.isEqual(prevResponse, httpResponseElement)
     )
 
     if not alreadyUsedRequest and not alreadyUsedResponse
       exampleIndex = exampleIndex + 1
       prevRequests = []
       prevResponses = []
 
-    # Check for empty http request
-    requestName = _.chain(httpRequest).get('meta.title', '').contentOrValue().value()
-    requestBody = trimLastNewline(if _.content(httpRequestBody) then _.content(httpRequestBody) else '')
-    requestSchema = trimLastNewline(if _.content(httpRequestBodySchemas) then _.content(httpRequestBodySchemas) else '')
-    requestAuthSchemes = transformAuth(httpTransaction, options)
-
-    requestHeaders = getHeaders(httpRequest)
-    requestHeaders1A = getHeaders1A(httpRequest)
-
     if not alreadyUsedRequest
       request = new blueprintApi.Request({
-        name: requestName
+        name: httpRequest.title?.toValue() or ''
```

Comment:
title will always be there (as empty string if not set). So this can just be `httpRequest.title.toValue()`